model/trace/messaging.yaml

```
@@ -15,16 +15,16 @@ groups:
           The [conversation ID](#conversations) identifying the conversation to which the message belongs,
           represented as a string. Sometimes called "Correlation ID".
         examples: 'MyConversationId'
-      - id: message.payload_size_bytes
+      - id: message.envelope.size
         type: int
         brief: >
-          The (uncompressed) size of the message payload in bytes.
-          Also use this attribute if it is unknown whether the compressed or uncompressed payload size is reported.
+          The size of the uncompressed message body and metadata in bytes.
         examples: 2738
-      - id: message.payload_compressed_size_bytes
+      - id: message.body.size
         type: int
-        brief: 'The compressed size of the message payload in bytes.'
-        examples: 2048
+        brief: >
+          The size of the uncompressed message body in bytes.
```

I believe the client library/instrumentation does not really know if body is compressed or not. Perhaps we can say something like 
```suggestion
          The size of the message (as passed by application code) body in bytes.
```

Or add a note saying that if SDK applies any compression, it should represent the size before compression is done.